src/plugins/summaryWidget/src/input/ColorPalette.js

```diff
@@ -33,17 +31,26 @@ function (
 
         this.palette.setNullOption('rgba(0,0,0,0)');
 
-        const domElement = $(this.palette.getDOM());
+        let domElement;
         const self = this;
 
-        $('.c-button--menu', domElement).addClass('c-button--swatched');
-        $('.t-swatch', domElement).addClass('color-swatch');
-        $('.c-palette', domElement).addClass('c-palette--color');
+        if (document.readyState === "complete"
```

Comment:
Hmm I remember getting an error when I had initially change this, but it must have been fixed as I went through and removed the rest of the zepto references since it works now. So, the `this.palette.getDOM()` function was initially wrapped like `$(this.palette.getDOM())`, which I believe is equivalent to calling the function on `document.ready()`. So, I mimicked it here. It does seem to work fine now when I just call `this.palette.getDOM()`, so I can remove it as well as the others!